agent/src/beerocks/slave/agent_db.cpp

```diff
@@ -96,4 +96,22 @@ void AgentDB::erase_client(const sMacAddr &client_mac, sMacAddr bssid)
     }
 }
 
+bool AgentDB::get_mac_by_ssid(const std::string &ruid, const std::string &ssid, std::string &value)
```

Comment:
Use sMacAddr instead of std::string
Use an explicit output parameter to hold error message if needed